### PR TITLE
Losing filled data when screen rotate (#3973)

### LINKED ISSUE #3973: Losing filled data when screen rotate
**Summary:** 

In signup  form filled information is lossed when screen is rotated 

**Steps to reproduce:** 
Go to Login activity 
click on signup
Fill signup  form  then rotated the screen 
  
What did you expect the app to do, and what did you see instead?
In signup  form filled information is not lossed when screen is rotated 

**Device and Android version:**
Device :Redmi Y3;

**Screen-shots:** 

WhatsApp Video 2020-10-18 at 8 04 15 PM (1)

**Would you like to work on the issue?**
Yes

Please let us know whether you want to fix the issue by yourself. If not, anyone can get the issue assigned to them.
i want to fix this issue

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
**Description (required)**

Fixes #3973 
What changes did you make and why?

**_Bug_** is due to the restart of SignupActivity  on the configuration change 
So  ****Add an attribute android:configChanges="screenSize|orientation|screenLayout" in   Manifes.xml** of auth.SignupActivity** so that SignupActivity does not restart on configuration change.

**Tests performed (required)**
Tested prodDebug on samsung m11 API level 29

Screenshots (for UI changes only)
Not losing filled data when screen rotate

### CODE REVIEW

**Review by ashishkumar468 — APPROVED** [review]
Please fix the formatting, looks good to me in general

**Comment by Prince-kushwaha** [comment]
ok

**Comment by Prince-kushwaha** [comment]
fixed the formatting

**Comment by Prince-kushwaha** [comment]
see change and merge the code

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -94,6 +94,7 @@
 
         <activity
             android:name=".auth.SignupActivity"
+          android:configChanges="orientation|screenLayout|screenSize"
```
Comment (ashishkumar468) [inline app/src/main/AndroidManifest.xml:97]: Hi, Please fix the formatting changes over here. Happy to merge post this

Comment (Prince-kushwaha) [reply app/src/main/AndroidManifest.xml:97]: I am beginner to open source contribution I does not  understand what  formatting changes make over above ?

**Review by ashishkumar468 — APPROVED** [review]

**Comment by Prince-kushwaha** [comment]
IMG_20201121_211418

what is meaning of All checks have failed and how solve this issue
**Please help me to solve this issue**

**Pull request merged**